addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionPluginUnitTest.java

```diff
@@ -56,9 +58,9 @@ protected int getRecommendMaxNumberMessagesPerParam(AttackStrength strength) {
             default:
                 return recommendMax + 20;
             case HIGH:
-                return recommendMax + 22;
+                return recommendMax + 27;
```

Comment:
Hi @kingthorin 
Actually for Command Injection Plugin, I have just added payloads to existing payloads and all the logic to run number of requests per param at different level of strength is same.

New payloads for only insane level are only added to TestPathTraversalUnitTest.java because they are just concatenation of extensions and null bytes to existing payloads.

Reply:
It might be better to do it just on the last level, no point keeping adding even more requests?

Reply:
actually i had just added more payloads and i have not changed the maximum number of request in High level, its just that there were less than 24 payloads for windows system so therefore the count has increased.

Reply:
OK.